Detections/SigninLogs/FailedLogonToAzurePortal.txt

```diff
@@ -0,0 +1,21 @@
+// Name: Failed login attempts to Azure Portal
+//
+// Description: This query over Azure AD sign-in activity highlights failed login attempts to Azure Portal 
+//
+// DataSource: #SigninLogs
+//
+// Techniques: #InitialAccess
+let timeRange = ago(30d);
```

Comment:
if it's a detection, and we are going to run it every day, then only look back 1 day?

Reply:
Made some additional changes, will push shortly.